manifest.json

```diff
@@ -311,7 +301,7 @@
         "rootDir": "fontawesome-free-5.9.0-web",
         "targetDir": "fontawesome-free-5.9.0-web",
         "bundle": {
-          "css": ["css/all.css"],
+          "css": ["css/fontawesome.css", "css/brands.min.css", "css/solid.min.css"],
```

Comment:
We use only brands and solid types of FA icons, the other types are in the paid version of FA.